Server/Server.Exec/MainServer.cpp

```diff
@@ -5,18 +5,12 @@
 #include "ArgParser.hpp"
 
 using Server::Builder::ServerBuilder;
+using Server::Builder::SettingsManager;
 
 int main(int argc, const char** argv)
 {
     ArgParser parser(argc, argv);
-    auto server = ServerBuilder()
-                                 .setValue(parser.getPair("--serverport"))
-                                 .setValue(parser.getPair("--dbname"))
-                                 .setValue(parser.getPair("--hostaddr"))
-                                 .setValue(parser.getPair("--port"))
-                                 .setValue(parser.getPair("--user"))
-                                 .setValue(parser.getPair("--password"))
-                                 .makeServer();
+    auto      server = ServerBuilder(std::make_unique<SettingsManager>(parser.GetSettings())).MakeServer();
```

Comment:
There are nop sence for extra make_unique. GetSettings return smart pointer

Reply:
Yeah, but parser.GetSettings() returns a std::unique_ptr<Server::Builder::Settings> for SettingsManager constructor and std::make_unique is colled for this SettingsManager.
It's correct.